packages/medusa/src/api/routes/admin/price-lists/update-price-list.ts

```diff
@@ -81,6 +84,9 @@ import { EntityManager } from "typeorm"
  *                 id:
  *                   description: The id of a customer group
  *                   type: string
+ *        includes_tax:
+ *             description: "[EXPERIMENTAL] Is the price list tax inclusive or not"
```

Comment:
suggestion: Rephrase to improve semantics
```suggestion
 *             description: "[EXPERIMENTAL] Tax included in prices of price list"
```